docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
     volumes:
       - ./.migrate:/home/node/app/.migrate
       - ./config:/home/node/app/config
+      - ./pdf:/home/node/app/pdf/
```

Comment:
@ketanreliefapps I think it's fine for debugging but we should remove it before mergin